hs_core/management/commands/solr_versions.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Debug discovery code on actual resources.
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+
+
+def check_displayed(resource, isreplacedby, replaces):
```

Comment:
Would it be valuable to also include assertions that discover matches what is found in this method?

Reply:
I added to the comment to this effect.